frontend/src/component/ai/AIChat.tsx

```diff
@@ -153,7 +180,7 @@ export const AIChat = () => {
                 minSize={{ width: '270px', height: '200px' }}
                 maxSize={{ width: '90vw', height: '90vh' }}
                 defaultSize={{ width: '320px', height: '450px' }}
-                onResize={scrollToEnd}
+                onResize={() => scrollToEnd({ onlyIfAtEnd: true })}
```

Comment:
Should we always scroll to end when resizing? I mean if you scroll up on a conversation and then resize...

Reply:
That's why we have `onlyIfAtEnd: true`. This is actually an improvement on: https://github.com/Unleash/unleash/pull/8456#discussion_r1802461279